thoth-precommit/Pipfile

```diff
@@ -9,4 +9,4 @@ verify_ssl = true
 pre-commit = "*"
 
 [requires]
-python_version = "3.6"
+python_version = "3.9"
```

Comment:
if we change this, do we need to `pipenv relock` ?

Reply:
You'll note that `Pipfile.lock` and `requirements.txt` are both updated (from running `pipenv lock` and `pipenv lock -r > requirements.txt`).